lib/utils/helpers.ts

```diff
@@ -406,6 +378,40 @@ export async function which(
 	return programPath;
 }
 
+/**
+ * Call which(programName) and spawn() with the given arguments.
+ * Reject the promise if the process exit code is not zero.
+ */
+export async function whichSpawn(
```

Comment:
This function is mainly to reduce boilerplate error handling associate with `spawn`, including checking the child process exit code.